DiceKeys/Views/Screens/DiceKeyWithDerivedValue.swift

```diff
@@ -178,10 +178,10 @@ extension FormCard where TitleContent == Text {
 
 struct DiceKeyWithDerivedValue_Previews: PreviewProvider {
     static var previews: some View {
-//        NavigationView {
+        NavigationView {
```

Comment:
We're getting rid of all uses of NavigationView to use our own version.

Reply:
It was commented for the preview, Debugging to get estimate or rough size of the screen. I did forgot to comment it back, reverting and pushing the code again.